erpnext_autoinstall/tests/test_commands.py

```diff
@@ -153,3 +154,29 @@ def test_set_user_role_profile(self):
         _set_user_role_profile(self.test_user_name, self.test_role_profile)
         self.assertEqual(frappe.get_doc(
             'User', self.test_user_email).role_profile_name, self.test_role_profile)
+
+    def test_get_user_api_secret_does_not_exists(self):
+        if frappe.__version__[:2] != "10":
+            api_secret = _get_user_api_secret("Alien42")
+            self.assertIsNone(api_secret)
+
+    def test_get_user_api_secret_admin(self):
+        if frappe.__version__[:2] != "10":
+            api_secret = _get_user_api_secret("Administrator")
+            self.assertIsNotNone(api_secret)
+
+    def test_add_user_api_key_admin(self):
+        if frappe.__version__[:2] != "10":
```

Comment:
Why ?

Reply:
API KEY doesn't exists in version 10

Reply:
ok, then mention this in the README documentation of the commands